app/models/work.rb

```diff
@@ -120,20 +120,30 @@ def publish_test_doi?
 
     private
 
-      def works_by_user_state(user, state)
-        # The user's own works by state (if any)
-        works = Work.where(created_by_user_id: user, state: state).to_a
+      def search_terms_where_clause(search_terms)
+        if search_terms.nil?
+          Work.where("true")
```

Comment:
@hectorcorrea I like .all better than .where("true"), but not going to hold up the PR for it...